VirtoCommerce.DynamicExpressionsModule.Web/Scripts/all-templates.html

```diff
@@ -55,13 +55,21 @@
 </script>
 <script type="text/ng-template" id="expression-ConditionInStockQuantity.html">
     InStock quantity is
-    <a class="__link" left-click-menu data-target="exactly_menu{{element1.id}}">{{element1.exactly | boolToValue:'exactly':'at least'}}</a>
+    <a class="__link" left-click-menu data-target="compareCondition_menu{{element1.id}}">{{element1.compareCondition | compareConditionToText}}</a>
```

Comment:
There is no backward compatibility support if the old expression has the `True` value for `Exactly`, this old value must be interpreted to the `Exactly`  and now it will be `at least`